src/components/form/radio-buttons/radio-button/radio-button.props.ts

```diff
@@ -11,7 +11,7 @@ export interface RadioChangeEvent {
 }
 
 export interface RadioButtonProps
-  extends MakeRequired<FormInputControlData<string, RadioChangeEvent>, 'name'>,
+  extends ReplacePropertyType<HeadlessRadioButtonProps, 'onChange', (value?: RadioChangeEvent) => void>,
```

Comment:
Why was the extends changed here? OK after looking at your other changes I think I know why you think you needed to do this, but you should leave the `onChange` with the headless type in it.